source/scenes/home/Asset/TxsPanel/TxsPanel.container.tsx

```diff
@@ -35,16 +35,19 @@ const TxsPanelContainer: FC<ITxsPanel> = ({ address, transactions }) => {
     if (!txHash) {
       return null;
     }
+    const DAG_OBJECT = DAG_NETWORK[activeNetwork.Constellation];
 
-
-    let DAG_EXPLORER = DAG_NETWORK[activeNetwork.Constellation].explorer;
+    if (!DAG_OBJECT) return '';
```

Comment:
Would this effect the user when they click the explorer icon in the top right of the asset screen?
Ex. The user clicked the explorer icon in the top right and since there is no link the user is navigated to a blank page?